README.md

```diff
@@ -1,79 +1,142 @@
-# nginx-k8s-loadbalancer
+<div>
+    <span>
+<img src="nkl-logo.svg" width="124" />
+</span>    
+<span>
+<h2>nginx-k8s-loadbalancer</h2>
+</span>
+</div>
 
-## Welcome to the Nginx Kubernetes Load Balancer project !
+The NGINX K8s Loadbalancer, or _NKL_, is a Kubernetes controller that provides TCP load balancing external to a Kubernetes cluster running on-premise.
 
-<br/>
+## Requirements
 
-This repo contains source code and documents for a new Kubernetes Controller, that provides TCP load balancing external to a Kubernetes Cluster running On Premises.  
+### Who needs NKL?
 
-<br/>
+- [ ] If you find yourself living in a world where Kubernetes is running on-premise instead of a cloud provider, you might need NKL.
+- [ ] If you want exceptional, best-in-class load-balancing for your Kubernetes applications, you might need NKL.
```

Comment:
I know a lot of open source projects do this, but I'd wait to add it in until we have some production evidence for it.